eng/tox/create_wheel_and_install.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import print_function
+from subprocess import check_call
+import argparse
+import os
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description = 'Build a package directory into a wheel. Then install it.')
+    parser.add_argument(
+        '-d',
+        '--distribution-directory',
+        dest='distribution_directory',
+        help='The path to the distribution directory. Should be passed $(Build.ArtifactStagingDirectory) from the devops yaml definition.',
+        required=True)
+
+    parser.add_argument(
+        '-p',
+        '--path-to-setup',
+        dest='target_setup',
+        help='The path to the setup.py (not including the file) for the package we want to package into a wheel and install.',
+        required=True
+        )
+
+    args = parser.parse_args()
+
+    check_call(['python', os.path.join(args.target_setup, 'setup.py'), 'bdist_wheel', '--universal', '-d', args.distribution_directory])
```

Comment:
If we ever were to include non-universal wheels (e.g. uamqp) in our repo, this would be broken. Do we have a place somewhere that lists assumptions our engineering system is making? I'm not saying that it needs to be addressed now, but it may be beneficial to have a list to avoid incorrect assumptions in the future.

Reply:
1) You're 100% correct. This was intentional.
2) Engineering Assumptions. That's a great suggestion and I'll get started on one.